Packs/Thycotic/Integrations/Thycotic/Thycotic.yml

```diff
@@ -0,0 +1,1096 @@
+      name: enabled
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether Duo two-factor authentication is enabled
+      isArray: false
+      name: fido2twofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: IsApplicationAccount
+      isArray: false
+      name: isapplicationaccount
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether OATH two-factor authentication is enabled
+      isArray: false
+      name: oathtwofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether RADIUS two-factor authentication is enabled
+      isArray: false
+      name: radiustwofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: RADIUS username
+      isArray: false
+      name: radiususername
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether two-factor authentication is enabled
+      isArray: false
+      name: twofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    deprecated: false
+    description: Create a new user
+    execution: false
+    name: thycotic-user-create
+    outputs:
+    - contextPath: Thycotic.User.Create
+      description: User object
+      type: String
+  - arguments:
+    - default: false
+      description: Filter users by Active Directory domain (integer)
+      isArray: false
+      name: filter_domainid
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether to include inactive users in the results
+      isArray: false
+      name: filter_includeinactive
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: Fields to search
+      isArray: false
+      name: filter_searchfields
+      required: false
+      secret: false
+    - default: false
+      description: Search text
+      isArray: false
+      name: filter_searchtext
+      required: false
+      secret: false
+    - default: false
+      description: Number of records to skip before taking results
+      isArray: false
+      name: skip
+      required: false
+      secret: false
+    - default: false
+      description: Sort direction
+      isArray: false
+      name: sortBy[0]_direction
+      required: false
+      secret: false
+    - default: false
+      description: Sort field name
+      isArray: false
+      name: sortBy[0]_name
```

Comment:
This (and similar) are not valid names for arguments, let's fix them

Reply:
@guykeller 
Replaced variable names